get total expenses by day/month/year in a period of time

schiaudoriana opened the pull request:
Retrieve statistics of expenses grouped by day/month/year, for a given period of time and for a specific category (or All categories)
Added a new tab inside the Statistics view:

![image](https://user-images.githubusercontent.com/79404760/144753712-1eb0454f-8fc5-4bb3-bd03-b5ca14dd80ec.png)

schiaudoriana committed:
get total expenses by day/month/year in a period of time

Tonirel reviewed (approved):
Graph looks really good!
What errors do you get when building?

adipopbv commented:
The build is failing because the maximum css budgets of some files (from what i can tell 4kB) have been exceeded because of importing some external css in those files. To fix that, you should either make the css smaller or change the warning and error threshold from the `angular.json` config: 
```
...
"budgets": [
	...
	{
		"type": "anyComponentStyle",
		"maximumWarning": "2kb",
		"maximumError": "4kb"
	}
],
...
```

to something like 200kB to be safe.

schiaudoriana committed:
removed css bootstrap import + some minor fixes to doughnut chart dimensions

schiaudoriana committed:
changed initial buget to 2mb

schiaudoriana commented:
addressed. thank you 
so i removed the unused css bootstrap imports, at first i thought they were necessary for the charts but turned out they were not 😆 after doing that i still had an error build telling me that i have exceeded the initial budget so i upped it to 2mb.

 "budgets": [
                {
                  "type": "initial",
                  "maximumWarning": "2mb",
                  "maximumError": "2mb"
                },
